README.md

```diff
@@ -1,39 +1,120 @@
-# Glowing Spoon
+# Glowing Spoon Chatter Box - Frontend for Llama 3
 
-## Introduction
+## Application Version: 1.1
+## Developed by: Sauraav Jayrajh
 
-This program is designed to help you learn the skills needed to apply the DevOps principles to your everyday work going forward in your career.
-Learning from leaders in the market space, there are some very basic aspects that you can learn to help you get started.
+---
 
-## Prerequisites
+## Developer Contact Details: 
 
-- Basic understanding of programming
-- Basic understanding of the command line
-- Basic understanding of git
-- A computer with internet access
-- A GitHub account
+- **Phone:** 071 924 4175
```

Comment:
Very noble, just be careful with adding personal details directly into a public repository 😉